### ISSUE TITLE
Conflict with MIA

### ISSUE THREAD

**Issue by Krutoy242** [Invadermonky/JustEnoughMagiculture#1]
Im developer of [E2E-E](https://www.curseforge.com/minecraft/modpacks/enigmatica-2-expert-extended) modpack. Last version i added JEM into it.

When JEM and [Minor Integrations and Additions (MIA)](https://www.curseforge.com/minecraft/mc-mods/minor-integrations-and-additions-mia) installed, some HEI (JEI) tabs looks strange.

For now, I found only one bug - IceAndFire mobs are doubled but some of them have default villager model instead. I tested only 1. MIA 2. MIA+JEM. I didnt tested JEM alone so cant proof that Farmer villager model will appear on only JEM.

![image](https://github.com/Invadermonky/JustEnoughMagiculture/assets/936618/ef9ba840-3cb9-4a25-9b57-0fa5c55e4534)

**Comment by Invadermonky** [comment]
### TLDR;
Both JEM and MIA have config options to disable the duplicate features.

### Longer Explanation:
JEM and MIA have overlap with anything added to JER. I designed JEM to replace and improve on many of MIA's JER integrations due to MIA crashing with many newer mod forks and having a number of hard-coded features.

Both JEM and MIA have configuration options that allow you to disable specific integrations (which would remove the duplicates) and JEM allows you disable specific mobs/villagers.

Other JEM/MIA overlaps may include:
- Botania
- Charm
- EnderIO
- FutureMC
- Harvestcraft
- Ice and Fire
- Industrial Foregoing
- Quark
- Thermal Foundation
- Thaumcraft

As for the Farmer Villager model showing, both MIA and JEM modify the default JER villager JEI display (in nearly identical ways) to support different villager models. However, due to a weird bit of code in JER, all custom villagers are removed prior to JEI registry and JEI loading is non-deterministic.

To remedy this issue, JEM uses ASM to guarantee proper load order, whereas MIA uses the "I hope it works" method.

**Comment by Krutoy242** [comment]
> Other JEM/MIA overlaps may include:
> 
> * Botania
> * Charm
> * EnderIO
> * FutureMC
> * Harvestcraft
> * Ice and Fire
> * Industrial Foregoing
> * Quark
> * Thermal Foundation
> * Thaumcraft


Great to know!
In this case I will disable all MIA=>JER integrations for all this mods and see if any other problems occur.

**Comment by Krutoy242** [comment]
Oh, nice! Now render even better and Thaumcraft mobs, for example, have armor! 

![image](https://github.com/Invadermonky/JustEnoughMagiculture/assets/936618/b607820f-b174-41c0-9f7d-5e7be51f0af0)
